core/src/test/java/com/ibm/wala/core/tests/cha/AnalysisScopeTest.java

```diff
@@ -55,4 +62,75 @@ public void testBaseScope() throws IOException, ClassHierarchyException {
                 ClassLoaderReference.Application, "Ljava/awt/AlphaComposite")),
         "found unexpected class");
   }
+
+  @Test
+    public void testToJson() throws IOException {
+        AnalysisScope scope =
+            AnalysisScopeReader.instance.readJavaScope(
+                TestConstants.WALA_TESTDATA,
+                // new FileProvider().getFile("J2SEClassHierarchyExclusions.txt"),
+                new FileProvider().getFile("GUIExclusions.txt"),
+                AnalysisScopeTest.class.getClassLoader());
+        Gson gson = new Gson();
+        Type type = new TypeToken<LinkedHashMap<String, Object>>(){}.getType();
+        LinkedHashMap<String, Object> map = gson.fromJson(scope.toJson(), type);
+        System.out.println(map);
+        if(map.containsKey("Exclusions")) {
+            String[] exclusions = scope.getExclusions().toString().split("\\|");
```

Comment:
Here we know what is in `GUIExclusions.txt`.  So I think we can simplify this code.  We can just do something like `assertEquals(List.of("java\/awt\/.*", "javax\/swing\/.*", "sun\/awt\/.*", "sun\/swing\/.*"), map.get("Exclusions"))`.  We can then get rid of this whole loop that re-parses the exclusions string